domains/api/http/transport.go

```diff
@@ -90,11 +90,11 @@ func MakeHandler(svc domains.Service, authn authn.Authentication, mux *chi.Mux,
 		r.Route("/{domainID}/invitations", func(r chi.Router) {
 			r.Use(api.AuthenticateMiddleware(authn, true))
 			r.Post("/", otelhttp.NewHandler(kithttp.NewServer(
-				sendInvitationEndpoint(svc),
+				sendDomainInvitationEndpoint(svc),
```

Comment:
Please rename the `decodeSendInvitationReq` to `decodeSendDomainInvitationReq` and its related Response struct